extensions/websockets/client/deployment/src/main/java/io/quarkus/websockets/client/deployment/WebsocketConfig.java

```diff
@@ -1,24 +1,26 @@
 package io.quarkus.websockets.client.deployment;
 
-import io.quarkus.runtime.annotations.ConfigItem;
 import io.quarkus.runtime.annotations.ConfigPhase;
 import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
 
 @ConfigRoot(phase = ConfigPhase.BUILD_TIME)
-public class WebsocketConfig {
+@ConfigMapping(prefix = "quarkus.websocket")
+public interface WebsocketConfig {
```

Comment:
@gsmet do not approve yet... because it

Reply:
Should be `quarkus.websocket`? I am not sure

Reply:
Thank you! I forgot this all-config page <3

Reply:
np!